### PR TITLE
Test cases for schemastore

### PR DESCRIPTION
**Pull request by KartikKapil**
will be needing test cases for this .JSON to make sure the schema gets uploaded on Schemaster 

Ref  #6363

### CODE REVIEW

File: readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "method": "enum('pip', 'setuptools', required=False)",
+    "extra_requirements": "list(str(), required=False)"
+  },
+  "python-install-pipfile": {
+    "pipfile": "path()",
+    "dev": "bool(required=False)",
+    "ignore_pipfile": "bool(required=False)",
+    "skip_lock": "bool(required=False)"
+  }
+}
```
Comment (stsewd) [inline readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json]: This format isn't the one used by json schema, you should find the appropriate equivalents of the spec in yamale for json schema

Comment (KartikKapil) [reply readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json]: @stsewd i already have the yaml file should i upload the same to Schemaster , i would still be needing the test cases of the following , also what possible changes should i make in the given schema

Comment (stsewd) [reply readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json]: @KartikKapil you should upload the json schema here so we can review it.

Comment (KartikKapil) [reply readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json]: @stsewd alright thank you, will try to upload it asap.

**Pull request closed without merging**